modules/external-api/src/main/java/org/opencastproject/external/endpoint/GroupsEndpoint.java

```diff
@@ -154,7 +161,16 @@ public Response getGroup(@HeaderParam("Accept") String acceptHeader, @PathParam(
                   @RestResponse(description = "The specified group does not exist.", responseCode = HttpServletResponse.SC_NOT_FOUND) })
   public Response deleteGroup(@HeaderParam("Accept") String acceptHeader, @PathParam("groupId") String id)
           throws NotFoundException {
-    return indexService.removeGroup(id);
+    try {
+      indexService.removeGroup(id);
+      return Response.noContent().build();
+    } catch (NotFoundException e) {
+      return Response.status(SC_NOT_FOUND).build();
+    } catch (UnauthorizedException e) {
+      return Response.status(SC_FORBIDDEN).build();
+    } catch (Exception e) {
+      throw new WebApplicationException(e);
```

Comment:
I had a quick look at other endpoints of the external API and found that this most often is used like:


```
import static javax.servlet.http.HttpServletResponse.SC_INTERNAL_SERVER_ERROR;

[...]
 logger.error("Unable to delete group {}", id, e);
 throw new WebApplicationException(e, SC_INTERNAL_SERVER_ERROR);

```

Reply:
done